docs/Informatiemodellen.md

```diff
@@ -0,0 +1,19 @@
+# Informatiemodellen
+
+Hier vind je de specificaties van de door VNG Realisatie beheerde informatiemodellen, gericht op informatiearchitecten en ontwikkelaars die met deze modellen werken.
+
+Als je meer wilt weten over informatiemodellen en welke van belang zijn voor gemeenten, kun je terecht op [GEMMA online](https://redactie.gemmaonline.nl/index.php/Wat_is_een_informatiemodel). Daar vind je een lijst van alle voor gemeenten relevante informatiemodellen, ook die niet door VNG Realisatie worden onderhouden.
+
+## Wat is een informatiemodel?
+Een informatiemodel is een gestructureerde beschrijving van de structuur, semantiek en eigenschappen van informatie over entiteiten in de werkelijkheid. Deze modellen bieden duidelijkheid over de betekenis en samenhang van concepten en dienen als basis voor het ontwikkelen van applicaties en systemen.

```

Comment:
Wellicht wat nuanceren. Je modelleert de structuur, semantiek en eigenschappen van de voor het betreffende domein benodigde informatie over entiteiten. Je modelleert immers niet alle eigenschappen. Voor de BRP is het immers niet van belang om van een persoon zijn oogkleur vast te leggen. Op dit moment i.i.g. niet... 😉